build/tools/cosmovisor.md

```diff
@@ -0,0 +1,55 @@
+# Cosmovisor
+
+_This doc is to be reviewed and updated_
+
+**cosmovisor** is a small process manager for any Cosmos SDK based application binaries that monitors the governance module for incoming **chain upgrade proposals**. If it sees a proposal that gets approved, cosmovisor can automatically download the new binary, stop the current binary, switch from the old binary to the new one, and finally restart the node with the new binary.
+
```

Comment:
Lets link here to the doc in which we recommend node operators to use cosmovisor for upgrades: https://docs.persistence.one/build/nodes-and-endpoints/join-testnet#cosmovisor

Let's also make it clear below that we don't recommend the auto-download feature for validators (they are supposed to check the new binaries), but that this is more targetted towards non-validator nodes.

Reply:
Actually, I have linked cosmos-sdk cosmovisor doc just after this passage to point to detailed information. And this document states the same warning as you have mentioned:-

But no issues, I will link to Persistence- cosmovisor doc instead.

Reply:
Let's link to both? 
Link to persistence docs to show how to use cosmovisor specifically for persistence

Link to cosmos-sdk docs to see how cosmovisor works more generically

Reply:
I hope, updated doc on the existing pages resolves this point.